pkg/config/config.go

```diff
@@ -464,6 +473,8 @@ var DefaultConfig = Config{
 			{Mime: webrtc.MimeTypeAV1},
 		},
 		EmptyTimeout: 5 * 60,
+		// TODO: remove
+		EnablePSRPC: true,
```

Comment:
Is this `REMOVE` for before merging? Or for deprecation when this becomes the norm? Guessing with OSS projects, a full switch could be a while?